tests/unit/processor/datetime_extractor/test_date_time_extractor.py

```diff
@@ -134,6 +132,83 @@ def test_an_event_extracted_datetime_and_local_utc_without_delta(self):
         }
         assert document == expected
 
+    def test_deletes_source_field(self):
```

Comment:
This test doesn't run on my local machine. I think you have to add the following lines before the `self.object.process(document)` line, similar to the other tests. 

```
self.object._local_timezone = tzutc()
self.object._local_timezone_name = DatetimeExtractor._get_timezone_name(
    self.object._local_timezone
)
```

Reply:
done